docs/topics/git.md

```
@@ -15,4 +15,83 @@
+
+설명) 
+   fetch로 선택한 repository의 이력만을 가져온다.
+   pull 명령어는 fetch와 merge를 같이 수행해 주는 명령어라고 볼 수 있다.
+```
+---
+## git reset
+### 개념
+- commit 이력을 과거 특정 지점으로 돌려놓는다.
+
+예시\)
+<img src="image/selectOptionForResetPoint.png" width="460" height="250" />
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)